scripts/init_holders.template

```diff
@@ -1,17 +1,9 @@
-const web3 = require('web3')
+const web3 = require('web3');
 const init_holders = [
-  {
```

Comment:
Can we still keep these two address, becausee the default relayer account on dev-net and QA are these two.

Reply:
Sure. I was just wondering where those two addresses came from.